webdriver/tests/bidi/browsing_context/navigate/fragment_navigated.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+
+from tests.support.sync import AsyncPoll
+from webdriver.bidi.modules.script import ContextTarget
+
+from ... import any_int, any_string, recursive_compare
+
+pytestmark = pytest.mark.asyncio
+
+EMPTY_PAGE = "/webdriver/tests/bidi/browsing_context/navigate/support/empty.html"
```

Comment:
Please rebase to latest master given that https://github.com/web-platform-tests/wpt/pull/40134 moved those support files one level up.